src/platform/plugins/shared/dashboard/server/api/create/create.ts

```diff
@@ -22,15 +22,11 @@ export async function create(
 ): Promise<DashboardCreateResponseBody> {
   const { core } = await requestCtx.resolve(['core']);
 
-  const { references: incomingReferences, ...incomingDashboardState } = searchBody.data;
   const {
     attributes: soAttributes,
     references: soReferences,
     error: transformInError,
-  } = transformDashboardIn({
-    dashboardState: incomingDashboardState,
-    incomingReferences,
-  });
+  } = transformDashboardIn(searchBody.data);
```

Comment:
Why are we able to remove the references from here? Is this endpoint not used by the UI?